internal/runbits/runtime/requirements/uninstall.go

```diff
@@ -0,0 +1,219 @@
+package requirements
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/go-openapi/strfmt"
+
+	"github.com/ActiveState/cli/internal/constants"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/multilog"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/rtutils/ptr"
+	"github.com/ActiveState/cli/internal/runbits/rationalize"
+	"github.com/ActiveState/cli/internal/runbits/runtime"
+	"github.com/ActiveState/cli/internal/runbits/runtime/trigger"
+	"github.com/ActiveState/cli/pkg/buildscript"
+	"github.com/ActiveState/cli/pkg/localcommit"
+	"github.com/ActiveState/cli/pkg/platform/api/buildplanner/types"
+	"github.com/ActiveState/cli/pkg/platform/model"
+	bpModel "github.com/ActiveState/cli/pkg/platform/model/buildplanner"
+)
+
+func (r *RequirementOperation) Uninstall(requirements []*Requirement) (rerr error) {
```

Comment:
Like with installing platforms, this functionality should be in a separate area for readability and maintainability. Uninstalling requires more subtle code path differences from installing.

However, we can group requirements and platforms into the same function because there's only one code branch that's easy to follow.